.vscode/linenote/frontend/client-list/client-list.component.tsx#L96.md

```diff
@@ -0,0 +1 @@
+Use as example for calling dispatchApiState ActionType.fetch to refresh after deletion
```

Comment:
Please remove this file from source control and add it to the gitignore.

Add the following to the `.gitignore` file:
```
.vscode
```

Then run the following commands in a terminal:
```sh
git rm --cached .vscode
git status # the vscode files should now say it is going to be deleted
git add .
git commit
```

Reply:
![image](https://user-images.githubusercontent.com/44277473/64902859-a4ba5e00-d66c-11e9-9139-46563e799502.png)